packages/vue/package.json

```diff
@@ -25,11 +26,13 @@
   ],
   "devDependencies": {
     "@vue/test-utils": "^2.0.0-rc.6",
+    "change-case": "^4.1.2",
+    "fs-extra": "^10.0.0",
     "rimraf": "^2.6.3",
     "rollup": "^2.51.1",
     "rollup-plugin-terser": "^7.0.2",
     "typescript": "^4.3.2",
-    "vue": "^3.1.1"
+    "vue": "3.0.0"
```

Comment:
**remark**: needed for now to prevent type errors in [Volar](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar)